proposals/revised-governance.md

```diff
@@ -0,0 +1,332 @@
+  reflects the actual decision making processes followed.
+
+## Proposal
+
+What I am proposing is a revised TSC/CTC structure such that:
+
+1. The TSC/CTC are generally folded back into a single administrative body
+   called the TSC.
+   * A Core Working Group with responsibility for overseeing the core Node.js
+     project (nodejs/node) is chartered (this is essentially what the CTC is
+     today)
+   * A special Admin team with responsibility over budget allocations and other
+     administrative actions would be established.
+2. The individual voting member seats in the TSC would be eliminated. Each
+   chartered working group would be granted a single voting seat within the TSC.
+   When a vote is required, each working group decides for itself the process it
+   will follow for exercising its one vote. The only requirement is that when a
+   vote is to be registered, there must be public documentation that details how
+   the vote was determined (could be meeting minutes, for instance).
+3. The TSC meeting schedule would be moved to one TSC meeting per month, on a
+   rotating time schedule.
+   * The meeting may be in person or fully remote.
+   * The meeting would consist of a public portion only.
+   * Any documented member of any chartered working group may attend.
+   * Anyone who is not a current member of a chartered working group may attend
+     on invitation from a chartered Working Group member.
+   * The meeting would include a review of working group activities as opposed
+     to an individual stand-up.
+4. Most decisions are made within the working groups. Any working group member
+   may escalate any decision by any working group by tagging it as `tsc-review`
+   and at-mentioning the other working groups that should review that decision.
+   As much as possible, an attempt should be made to resolve the issue through
+   discussion on GitHub. If the tagged issue is not determined to be resolved
+   in time for the monthly TSC meeting, the TSC Chair (and only the TSC Chair)
+   may escalate it to `tsc-agenda` for discussion at the meeting.
+   * If it appears clear that consensus cannot be reached on an issue tagged
+     `tsc-agenda` through regular discussion on GitHub or at the TSC meeting,
+     any working group member may tag the issue as `tsc-vote`, which must be
+     seconded by at least one other working group member.
+   * Issues marked `tsc-vote` would go to a vote of chartered working groups.
+     Working group votes would be collected via GitHub comments on the issue,
+     with links to the documentation describing how the vote was reached. The
+     vote will be completed when either enough votes are received to consider
+     the matter decided, or by the next TSC meeting (whichever comes first).
+     Any votes not registered by the next TSC meeting would be considered to
+     be abstentions.
+5. New working groups would be chartered only through consensus or vote of all
```

Comment:
I don't know if it would make it any worse. The steps are all pretty much the same as they are today. The only change is on who votes for the final charter.